src/test/java/bio/terra/janitor/integration/TrackResourceIntegrationTest.java

```diff
@@ -262,6 +281,70 @@ public void subscribeAndCleanupResource_googleBigQueryTable() throws Exception {
     assertTrue(bigQueryCow.delete(datasetName));
   }
 
+  @Test
+  public void subscribeAndCleanupResource_googleProject() throws Exception {
+    String projectId = randomProjectId();
+
+    Operation operation =
+        resourceManagerCow
+            .projects()
+            .create(new Project().setProjectId(projectId).setParent(parentResourceId))
+            .execute();
+    OperationCow<Operation> operationCow = resourceManagerCow.operations().operationCow(operation);
+    operationCow =
+        OperationUtils.pollUntilComplete(
+            operationCow, Duration.ofSeconds(5), Duration.ofSeconds(30));
+    assertTrue(operationCow.getOperation().getDone());
+    assertNull(operationCow.getOperation().getError());
+
+    Project project = resourceManagerCow.projects().get(projectId).execute();
+    assertEquals(projectId, project.getProjectId());
+    assertEquals("ACTIVE", project.getLifecycleState());
+
+    CloudResourceUid resource =
+        new CloudResourceUid().googleProjectUid(new GoogleProjectUid().projectId(projectId));
+
+    publishAndVerifyResourceTracked(resource);
```

Comment:
I'm not sure from just the function name what's being verified. maybe `publishAndVerifyCleanupDone`?